.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -29,7 +29,7 @@ body:
       description: >-
         A clear and concise description of what actually happened instead.
         Was the output confusing or poorly described? Please provide steps to reproduce this bug.
-      placeholder: |
+      value: |
```

Comment:
I guess here there is nothing to copy & paste, so perhaps placeholder is appropriate??

Reply:
Actually, never mind: I just realized that of course I can now *try* making a bug report and I think for this one would not want to copy/paste, but it seems nice to have the 1,2,3 and python example set up so one can just edit them. So, this would be good.